src/backend/hook.ts

```diff
@@ -10,11 +10,17 @@ export const useJobAPI = () => {
   /**
    * @returns Promise job id
    */
-  const submitJob = async (
-    // TODO: fix invalid oas schema (invalid fields: status, created_at, updated_at)
-    job: JobsSubmitJobRequest
-  ): Promise<string /* job id */> => {
-    return api.job.submitJob(job).then((res) => res.data.job_id);
+  const submitJob = async (job: JobsSubmitJobRequest): Promise<string /* job id */> => {
+    return api.job
+      .submitJob({
+        ...job,
+        // TODO: fix invalid oas schema (generated types)
+        // `transpiler_info`, `simulator_info` and `mitigation_info` expect JSON formatted string
```

Comment:
I'm sorry I didn't tell you, but the OAS schema is changed and 
transpiler_info, mitigation_info, and simulator_info is now object with arbitrary properties, so you should not stringify these values.
For details, please see [the OAS schema](https://github.com/oqtopus-team/oqtopus-frontend/blob/f3f6c6c7530a0db39b152f7230ace3079c2923a9/src/api/generated/openapi.yaml#L1058).